chainer/functions/clipped_relu.py

```diff
@@ -0,0 +1,69 @@
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+import numpy
+
+
+class ClippedReLU(function.Function):
+
+    """clipped ReLU function. clipped ReLU is written as below,
+
+    :math:`ClippedReLU(x, z) = \min(\max(0, x) ,z)`
+    z is a parameter to cap return value of ReLU.
+
+    """
+
+    def __init__(self, z):
```

Comment:
Do you need to check the value of `z`? Is it need to be positive, or all values are acceptable?